src/main/java/frc/robot/subsystems/SwerveDrivetrain.java

```diff
@@ -69,6 +69,7 @@ private double getGyro() {
 
     @Override
     public void periodic() {
+        gyroValue = getGyro();
         frontLeftDriveCurrent = ((CANSparkMax)frontLeft.getSwerveMotor().getDriveMotor()).getOutputCurrent();
```

Comment:
All the Current stuff can be placed inside `if(CURRENT_DEBUG)`. It can be set to `true` is we want it in the competition, but this way we at least have some control over it

Reply:
done